### PR TITLE
Feature issue 482

### LINKED ISSUE #482: Clean code to match Python coding style
### Describe the bug
Currently, many parts of our code don't follow Python coding style. Now, please clean it.
Please also group and arrange the `import` lines.

### Objective
- Follow PEP8 as much as possible. For line length, it can be 120 chars (PEP8 suggests 79, but I think that number is only relevant for old display device, while all of us now use wide screen monitor).
- Don't leave unused "import", unused variable. This can be detected with _pyflakes_ tool.

### Tools
This work doesn't need much brain. So we should leave it for tools. I recommend  _pycodestyle_, _pyflakes_. They help you find problematic lines, but not fix for you. Other than that, please install appropriate plugins to your code editor to integrate the tools.

This is how you can do in VS Code:
- Install [Python](https://marketplace.visualstudio.com/items?itemName=ms-python.python) extension.
- Add this to settings:

```json
{
    "python.linting.flake8Enabled": true,
    "files.trimTrailingWhitespace": true,
    "files.insertFinalNewline": true,
    "files.trimFinalNewlines": true
}
```

### PR DESCRIPTION
**Pull request by bhadreshpsavani**
Fixes #482 

#### Checklist

-  [x] I have read the Contribution & Best practices Guide and my PR follows them.
-  [x] My branch is up-to-date with the Upstream master branch.
-  [ ] The unit tests pass locally with my changes
-  [ ] I have added tests that prove my fix is effective or that my feature works
-  [x] I have added necessary documentation (if appropriate)
-  [x] All the functions created/modified in this PR contain relevant docstrings.

#### Test Passing

- [ ] The SUSI Server must be building on the pi on bootup
- [ ] The hotword detection should have a decent accuracy
- [ ] SUSI Linux shouldn't crash when switching from online to offline and vice versa (failing as of now)
- [ ] SUSI Linux should be able to boot offline when no internet connection available (failing)

#### Short description of what this resolves:
This issue is for code cleaning and structuring the code

#### Changes proposed in this pull request:
* arranged import statements
* arranged comment messages and code according to the character limit
* all class requires two lines above it according to pylint so added accordingly
* some minor changes suggested by pylint tool in vscode

### CODE REVIEW

**Comment by bhadreshpsavani** [comment]
hello @stealthanthrax 
I have resolved the conflicts,
Thanks

File: main/states/led.py [hunk 1]
```diff
@@ -14,8 +14,10 @@
 class LED_COLOR:
 
     # Constants
-    MAX_BRIGHTNESS = 0b11111  # Safeguard: Set to a value appropriate for your setup
-    LED_START = 0b11100000  # Three "1" bits, followed by 5 brightness bits
+    # Safeguard: Set to a value appropriate for your setup
```
Comment (stealthanthrax) [inline main/states/led.py]: can revert these comments as these were before? These comments lose the descriptiveness when changed to this way.

File: main/ui/animators.py [hunk 2]
```diff
@@ -40,12 +39,20 @@ def draw(self, ctx, width, height):
             ctx.set_line_width(6)
             ctx.set_line_cap(cairo.LINE_CAP_ROUND)
             if i % 2 == 0:
-                ctx.move_to(width / 2 + i * 10, height / 2 + 3 - 8 * math.sin(self.tc + i))
-                ctx.line_to(width / 2 + i * 10, height / 2 - 3 + 8 * math.sin(self.tc + i))
+                ctx.move_to(
```
Comment (stealthanthrax) [inline main/ui/animators.py]: These changes involving mathematical operations look a bit weird when moved to individual lines. Can revert them to as they were?

File: main/states/led.py (hunk 1 above)
Comment (bhadreshpsavani) [reply main/states/led.py]: i have revert the comments

File: main/ui/animators.py (hunk 2 above)
Comment (bhadreshpsavani) [reply main/ui/animators.py]: I have revert them as shown in below commits, Kindly review it :)

File: main/hardware_components/rpi_wake_button.py [hunk 3]
```diff
@@ -1,14 +1,15 @@
...
+
 class RaspberryPiWakeButton(WakeButton):
 
     def __init__(self):
         super().__init__()
         GPIO.setmode(GPIO.BCM)
         GPIO.setup(17, GPIO.IN, pull_up_down=GPIO.PUD_UP)
-        GPIO.add_event_detect(17, GPIO.FALLING, callback=self.button_detected, bouncetime=300)
+        GPIO.add_event_detect(
+            17, GPIO.FALLING, callback=self.button_detected, bouncetime=300)
```
[the remark this answers is not in the document]
Comment (bhadreshpsavani) [reply main/hardware_components/rpi_wake_button.py]: I have made changes

**Pull request merged**